src/status_im2/contexts/quo_preview/bottom_sheet/bottom_sheet.cljs

```diff
@@ -0,0 +1,56 @@
+(ns status-im2.contexts.quo-preview.bottom-sheet.bottom-sheet
+  (:require [quo2.components.buttons.button :as button]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [re-frame.core :as re-frame]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor [{:label "Show handle:"
+                  :key   :show-handle?
+                  :type  :boolean}
+                 {:label "Backdrop dismiss:"
+                  :key   :backdrop-dismiss?
+                  :type  :boolean}
+                 {:label "Expendable:"
+                  :key   :expandable?
+                  :type  :boolean}
+                 {:label "Disable drag:"
+                  :key   :disable-drag?
+                  :type  :boolean}])
+
+(defn bottom-sheet-content []
+  [rn/view {:style {:justify-content :center
+                    :align-items     :center}}
+   [button/button {:on-press #(do (re-frame/dispatch [:bottom-sheet/hide]))} "Close bottom sheet"]
```

Comment:
do you mean here ? this doesn't work ?

Reply:
Oh I just tried, sheet is closing, but without any animations

Reply:
hm it should work

Reply:
yes, i changed event `:bottom-sheet/hide` to only toggle `:bottom-sheet/show?` state in :db. This way, the Navigation overlay doesn't get dismissed automatically. In the BottomSheet component, it will get dismissed.

But still, only in this case it doesn't work

Reply:
Dragging down BottomSheet or clicking on the backdrop triggers animation, clicking the button doesn't somehow...

Reply:
oh i just realized bottom sheet shouldn't be in the quo, because it relies on the app state, so it should be in status-im2.common

Reply:
i don't see that you use subscription, you have to use subscription to animate bottom sheet when we close it

Reply:
also we shouldn't have a preview for bottom sheet itself in quo preview, we have drawers and there is bottom sheet is used it can be previewed there

Reply:
Doesn't it use the `:bottom-sheet` subscription? Which holds `show?`, `view` and `options`?

Reply:
yes but you don't use it, it's in the props, but you use your own atom `show-bottom-sheet?`

Reply:
i use it in the first effect (`visible?`), which triggers the change of `show-bottom-sheet?`

Reply:
you need to use show? from the props

Reply:
`show?` is used here (src/status_im/ui/screens/bottom_sheets/views.cljs) for the prop `visible?`

Reply:
yeah confusing, but anyway :visible? is not used also, it should be used

Reply:
It is used in `src/quo2/components/bottom_sheet/view.cljs` in line 136. That's why i don't i understand, why the animation isn't triggered.